x-pack/test/api_integration/apis/metrics_ui/services.ts

```diff
@@ -32,15 +31,19 @@ export default function ({ getService }: FtrProviderContext) {
         packageVersion: version,
       });
     });
-    after(async () => apmSynthtraceKibanaClient.uninstallApmPackage());
+    after(async () => {
+      await apmSynthtraceKibanaClient.uninstallApmPackage();
+    });
 
     describe('with transactions', () => {
       before(async () =>
         synthtraceApmClient.index(
           generateServicesData({ from, to, instanceCount: 3, servicesPerHost: 3 })
         )
       );
-      after(async () => synthtraceApmClient.clean());
+      after(async () => {
```

Comment:
Not sure if this needed
```
after(async () => {
    await synthtraceApmClient.clean();
});

doesn't work?
after(() => {
  synthtraceApmClient.clean();
});
  
```

Reply:
I would expect the previous code to work, actually
```
after(async () => synthtraceApmClient.clean());
```

But since it seems to be flaky, I decided to try being more explicit and use `await`

>doesn't work?
> after(() => { synthtraceApmClient.clean(); });

that doesn't work because it's neither awaiting the promise nor returning it to jest.

This *could* work:

```
after(() => synthtraceApmClient.clean());
```